model/modelpb/ip.go

```diff
@@ -53,16 +54,14 @@ func Addr2IP(addr netip.Addr) *IP {
 	return ip
 }
 
+// IP2Addr converts a valid IP to netip.Addr.
 func IP2Addr(i *IP) netip.Addr {
```

Comment:
Needs a nil check here since the nil check was previously wrapped in `GetV4`:

```go
	if i == nil {
		return netip.Addr{}
	}
```

Reply:
Also, another test case in `TestIP2Addr` for nil IP would not hurt I think. Something like:

```go
		{
			name: "empty",
			ip:   nil,

			expectedAddr: netip.Addr{},
		},
```

Reply:
nice catch, that's what caught by test panic